test/functional/api/cas/statistics.py

```diff
@@ -0,0 +1,643 @@
+class CacheStats:
+    stats_list = [
+        "config_stats",
+        "usage_stats",
+        "inactive_usage_stats",
+        "request_stats",
+        "block_stats",
+        "error_stats",
+    ]
+
+    def __init__(self, stats):
+        try:
+            self.config_stats = CacheConfigStats(
+                stats["cache id"],
+                stats["cache size"],
+                stats["cache device"],
+                stats["core devices"],
+                stats["inactive core devices"],
+                stats["write policy"],
+                stats["eviction policy"],
+                stats["cleaning policy"],
+                stats["promotion policy"],
+                stats["cache line size"],
+                stats["metadata memory footprint"],
+                stats["dirty for"],
+                stats["metadata mode"],
+                stats["status"],
+            )
+        except KeyError:
+            pass
+        try:
+            self.usage_stats = UsageStats(
+                stats["occupancy"], stats["free"], stats["clean"], stats["dirty"]
+            )
+        except KeyError:
+            pass
+        try:
+            self.inactive_usage_stats = InactiveUsageStats(
+                stats["inactive occupancy"] if "inactive occupancy" in stats else None,
+                stats["inactive clean"] if "inactive clean" in stats else None,
+                stats["inactive dirty"] if "inactive dirty" in stats else None,
```

Comment:
If inactive stats are not present in dict, python will raise `KeyError`, which you already handled, so you can remove all `if` statements.